shell/platform/tizen/flutter_tizen_engine_test.cc

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/tizen/flutter_tizen_engine.h"
+#include "gtest/gtest.h"
+
+TEST(FlutterTizenEngineTestSimple, Create_Headless) {
+  flutter::FlutterTizenEngine* tizen_engine =
+      new flutter::FlutterTizenEngine(false);
```

Comment:
Could you wrap this file with the ::flutter::testing namespace instead?

```cpp
namespace flutter {
namespace testing {

...

}  // namespace testing
}  // namespace flutter
```

You will need to change `testing::Test` to `::testing::Test`.

Reply:
thanks!